### DISCUSSION TITLE
Build Fails when trying to run example app.

### DISCUSSION THREAD

**Discussion by jason18401** [WalletConnect/walletconnect-monorepo#736] [Developer Support]
Hello I seem to get buildtime errors when trying to run the exampleApp for V2. I've screenshotted the errors below. Any help would be appreciated.

I added the package url:
dependencies: [
    .package(url: "https://github.com/WalletConnect/WalletConnectSwiftV2", .branch("main")),
],
<img width="276" alt="Screen Shot 2022-01-30 at 12 27 59 AM" src="https://user-images.githubusercontent.com/11168337/151687924-a1809e7b-9fca-49c6-8b66-cc625b7f9a2d.png">

**Comment by llbartekll** [comment]
hey @jason18401 you are trying to override package. You should be able to build an app without adding a package with url. The sdk is already in cloned repo.
